### PR TITLE
CSI-1196 bump pycqa linters version

### PR DESCRIPTION
**Pull request by oriyarde**

### CODE REVIEW

**Review by matancarmeli7 — APPROVED** [review]
LGTM

File: controller/scripts/lint.ini [hunk 1]
```diff
@@ -25,9 +13,12 @@ disable=
  # easy:
  raise-missing-from, # W - happened only on Jenkins, add "from" after some of the "raise"
  redefined-builtin, redefined-outer-name, # W, W - colliding names: e.g. type, e.g. config
- no-self-use, # R - some functions should be made static
+ unspecified-encoding, # W - add explicit encoding="utf-8" argument when calling "open"
+ consider-using-from-import, # R - redundant "as" after "import"
+ consider-using-with, # R - use "with" instead of "acquire" + "release"
```
Comment (ArbelNathan) [inline controller/scripts/lint.ini:18]: do we have tickets to remove the `disable` ones?

Comment (oriyarde) [reply controller/scripts/lint.ini:18]: I opened CSI-4374

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**